tests/unit/ui/components/TestUserReview.js

```diff
@@ -66,14 +62,65 @@ describe(__filename, () => {
 
     expect(
       root
-        .find('p')
+        .find('.UserReview-body')
         .render()
         .find('br'),
     ).toHaveLength(1);
   });
 
-  it('hides ratings for replies', () => {
+  it('does not render an empty review body', () => {
+    const root = render({
+      review: _setReview({ ...fakeReview, body: undefined }),
+    });
+
+    expect(root.find('.UserReview-body')).toHaveText('');
+  });
+
+  it('can render a fallback for reviews without a body', () => {
+    const bodyFallback = 'some kind of placeholder';
+    const root = render({
+      bodyFallback,
+      review: _setReview({ ...fakeReview, body: undefined }),
+    });
+
+    expect(root.find('.UserReview-body')).toHaveText(bodyFallback);
+  });
+
+  it('only renders a fallback when the review body is empty', () => {
+    const body = 'This is an actual review';
+    const bodyFallback = 'some kind of placeholder';
+    const root = render({
+      bodyFallback,
+      review: _setReview({ ...fakeReview, body }),
+    });
+
+    expect(root.find('.UserReview-body')).not.toHaveText(bodyFallback);
+    expect(
+      root
+        .find('.UserReview-body')
+        .render()
```

Comment:
Why are we calling `render()` in this assert, but not in the assert above?

Reply:
Yes, it's a bit confusing. It's because one of the if branches renders content:

https://github.com/mozilla/addons-frontend/blob/50d026b5bf44a5c91f506ff823c51518f6743f0e/src/ui/components/UserReview/index.js#L41

...and the other branch renders a React node:

https://github.com/mozilla/addons-frontend/blob/50d026b5bf44a5c91f506ff823c51518f6743f0e/src/ui/components/UserReview/index.js#L43

I'm not sure what to do about it. We would be able to make the assertions consistent if Enzyme fixes this bug: https://github.com/airbnb/enzyme/issues/419